### PR TITLE
created initiative card

### PR DESCRIPTION
**Pull request by saumitra1993**

### CODE REVIEW

**Comment by saumitra1993** [comment]
Screenshot 2019-05-31 at 9 47 15 AM

**Comment by saumitra1993** [comment]
@webbyharsh do review this PR.

File: src/components/InitiativeCard.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from "react";
+import "../styles/InitiativeCard.css";
+
+export default function InitiativeCard(props){
+  return (
```
Comment (shmshrivastava) [inline src/components/InitiativeCard.jsx:6]: Can we (or should we) define PropTypes?

**Review by shmshrivastava — CHANGES_REQUESTED** [review]
No CSS file required?

File: src/components/pages/Landing/LandingSectionOne.js [hunk 2]
```diff
@@ -7,6 +8,11 @@ export default class LandingSectionOne extends Component {
   }
 
   render(){
-    return <h1>Stub for landing section two</h1>
+    return (
+      <div>
+        <h1>Stub for landing section two</h1>
+        <InitiativeCard img="/images/sevatiffin.jpeg" title="Seva Tiffin" description="Food is served at the several locations for the needy at Hospitals, Schools for underprivileged children, Old age Homes. If you wish to serve at any of these locations you can Sign Up for Seva. You can also buy a meal for the location which will be delivered to the needy by Volunteers." />
```
Comment (webbyharsh) [inline src/components/pages/Landing/LandingSectionOne.js:14]: Like we previously did for "LandingSectionTwo" part where the props data was actually taken from other file, we can do the same here. The props data insted of hardcoding it directly in the component we can import it

Comment (saumitra1993) [reply src/components/pages/Landing/LandingSectionOne.js:14]: Yeah, we would do that in this task - https://www.proso.io/tasks/5cbb316e76374c5262ecdb1f

This task involved just creating one child of `InitiativeGroup` component. I just needed a way to demo it. We can remove it once the `InitiativeGroup` component is created.

File: src/components/InitiativeCard.jsx (hunk 1 above)
Comment (saumitra1993) [reply src/components/InitiativeCard.jsx:6]: Yeah good point, let me add that.

**Comment by webbyharsh** [comment]
everything looks fine with this PR. Can we merge it?

**Comment by saumitra1993** [comment]
yes please

**Comment by webbyharsh** [comment]
this branch has been merged with the master

**Pull request merged**